src/test/rgw/CMakeLists.txt

```diff
@@ -229,3 +229,4 @@ add_executable(unittest_rgw_lua test_rgw_lua.cc)
 add_ceph_unittest(unittest_rgw_lua)
 target_link_libraries(unittest_rgw_lua ${rgw_libs})
 
+add_subdirectory(dbstore)
```

Comment:
This may not be needed if the test file is moved to src/rgw/store/dbstore/tests .